src/components/choropleth.js

```diff
@@ -72,13 +73,21 @@ function ChoroplethMap({
       const margin = {left: 0.02 * widthLegend, right: 0.02 * widthLegend};
       const barWidth = widthLegend - margin.left - margin.right;
       const heightLegend = +svgLegend.attr('height');
+      let title = 'Confirmed Cases';
+
+      if (type === 1) {
+        title = 'Confirmed Cases';
+      } else {
+        title = 'Case Density Per Million';
+      }
```

Comment:
```title = type === 1 ? 'Confirmed Cases' : 'Case Density Per Million' ;```

Reply:
Future proofs it in case we add multiple views on future such as "tests per million"